android/app/build.gradle

```diff
@@ -73,8 +73,8 @@ import com.android.build.OutputFile
  */
 
 project.ext.react = [
-    cliPath: "node_modules/haul/bin/cli.js",
-    entryFile: "index.js"
+  cliPath: "node_modules/haul/bin/cli.js",
+  entryFile: "index.js"
```

Comment:
I think the indentation here is broken

Reply:
Good catch. I'll fix it in qr scanner branch.